pkgs/top-level/python-packages.nix

```diff
@@ -7751,6 +7751,26 @@ in modules // {
     };
   };
 
+  pycallgraph = buildPythonPackage rec {
+    name = "pycallgraph-${version}";
+    version = "1.0.1";
+
+    src = pkgs.fetchurl {
+      url = mirror://pypi/p/pycallgraph/pycallgraph-1.0.1.tar.gz;
+      sha256 = "0w8yr43scnckqcv5nbyd2dq4kpv74ai856lsdsf8iniik07jn9mi";
+    };
+
+    doCheck = false;
```

Comment:
why are tests disabled? Please include the reason as a comment. We prefer not to have tests disabled.